tests/conftest.py

```diff
@@ -13,7 +15,8 @@ def debug(request: pytest.FixtureRequest) -> bool:
     return cast(bool, request.param)
 
 
-@pytest.fixture(autouse=True)
-def loop(event_loop: AbstractEventLoop, debug: bool) -> AbstractEventLoop:
+@pytest_asyncio.fixture(autouse=True)
```

Comment:
I think the regular fixture should work?
```suggestion
@pytest.fixture(autouse=True)
```